### PR TITLE
[latency predictor] Solve the bug of downloading predictors' files

### PR DESCRIPTION
**Pull request by zzjjay**
In multi process training, the original download method will repeatedly download files.

### CODE REVIEW

File: paddleslim/analysis/_utils.py [hunk 1]
```diff
@@ -200,9 +202,11 @@ def download_predictor(op_dir, op):
 
     op_path = os.path.join(op_dir, op + '_predictor.pkl')
     if not os.path.exists(op_path):
-        subprocess.call(
-            f'wget -P {op_dir} https://paddlemodels.bj.bcebos.com/PaddleSlim/analysis/{op_path}',
-            shell=True)
+        # To solve the 'SSL: certificate verify failed' error.
```
Comment (wanghaoshuang) [inline paddleslim/analysis/_utils.py]: 请规范注释

File: paddleslim/analysis/_utils.py [hunk 2]
```diff
@@ -200,9 +202,11 @@ def download_predictor(op_dir, op):
...
+        ssl._create_default_https_context = ssl._create_unverified_context
+        url = f'https://paddlemodels.bj.bcebos.com/PaddleSlim/analysis/{op_path}'
```
Comment (wanghaoshuang) [inline paddleslim/analysis/_utils.py]: 把链接或URL前缀声明为一个变量，放在当前文件前面。

Comment (zzjjay) [reply paddleslim/analysis/_utils.py]: done

File: paddleslim/analysis/_utils.py (hunk 1 above)
Comment (zzjjay) [reply paddleslim/analysis/_utils.py]: done

**Review by wanghaoshuang — APPROVED** [review]

**Pull request merged**